### PR TITLE
fix(@aws-amplify/interactions): Upgrading LexV2 client

### PR DESCRIPTION
**Pull request by elorzafe**
#### Description of changes

Upgrading Lex v2 client

#### Issue #, if available

#10284 

#### Description of how you validated changes
- Run react-native app in production mode

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/interactions/package.json [hunk 1]
```diff
@@ -85,6 +85,7 @@
 				"statements": 0
 			}
 		},
+		"browser": true,
```
Comment (stocaaro) [inline packages/interactions/package.json:88]: What does this do?

**Review by stocaaro — APPROVED** [review]
LGTM. Since its a testing change, seems decent if the tests are passing.

File: packages/interactions/__tests__/providers/AWSLexV2Provider-unit-test.ts [hunk 2]
```diff
@@ -71,12 +83,7 @@ const gzipBase64Json = async (dataObject: object) => {
 		const arrayBuffer = strToU8(objString);
 
 		// 3. gzip compress
-		const compressedData: Uint8Array = await new Promise((resolve, reject) => {
-			gzip(arrayBuffer, (err, data) => {
-				if (err) reject(err);
-				else resolve(data);
-			});
-		});
+		const compressedData: Uint8Array = gzip(arrayBuffer);
```
Comment (Samaritan1011001) [inline packages/interactions/__tests__/providers/AWSLexV2Provider-unit-test.ts:86]: Curious - Need not be async? Even for big `arrayBuffer`?

**Review by Samaritan1011001 — APPROVED** [review]
Just a question but looks good to me 👍 
Thank you.

**Pull request closed without merging**